src/libstd/sync/mod.rs

```diff
@@ -42,3 +42,18 @@ mod condvar;
 mod mutex;
 mod once;
 mod rwlock;
+
+/// Save power or switch hyperthreads in a busy-wait spin-loop.  This
+/// function is deliberately more primitive than
+/// `std::thread::yield_now` and does not directly yield to the
+/// system's scheduler.  In some cases it might be useful to use a
+/// combination of both functions.
+#[inline]
+#[stable(feature = "rust1", since = "1.1.6")]
```

Comment:
@sfackler Actually can you help me out with the stability attribute syntax? I can't find any reference of it online.